libbeat/processors/actions/docs/lowercase.asciidoc

```diff
@@ -0,0 +1,65 @@
+[[lowercase]]
+=== Lowercase from events
+
+++++
+<titleabbrev>lowercase</titleabbrev>
+++++
+
+The `lowercase` processor specifies a list of fields that must be lowercased. This applies only to keys that match the given fields. The default behaviour of the processor is to modify the most nested key in the chain. Use `full_path` to lowercase all keys in the nested chain.
+
+
+
+==== Examples: 
+
+[source,yaml]
+----
+processors:
+  - rename:
+      fields:
+        - "a.B"
```

Comment:
I think this processor only makes sense if the key comparison is case-insensitive. See the customer's use-case https://github.com/elastic/beats/issues/22254#issuecomment-722706018

We need to highlight this in the docs and adjust the implementation.

If we need so, we can introduce the `case_sensitive` flag in the config later.

```suggestion
        - "a.b.c.d"
```